tests/service/certificate/test_service_certificate.yml

```diff
@@ -76,7 +77,7 @@
       ok_as_delegate: no
       ok_to_auth_as_delegate: no
     register: result
-    failed_when: not result.changed
+    failed_when: not result.changed and result.failed
```

Comment:
Why is this "and result.failed",  shouldn't it be "or result.failed"?

Reply:
Same as before... also fixed.